### PR TITLE
SentinelOne v2: Fixes sentinelone-disconnect-agent and sentinelone-connect-agent issues.

### PR DESCRIPTION
**Pull request by chloerongier**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
This PR fixes issues regarding the **sentinelone-disconnect-agent** and **sentinelone-connect-agent** commands:
- The commands would only return the first Agent ID and its network status among the IDs given as inputs.
- The output *SentinelOne.Agent.AgentsAffected* of the **sentinelone-connect-agent** command is not implemented in the code.
- Missing definition in the yml of the output *SentinelOne.Agent.NetworkStatus* for the **sentinelone-connect-agent** command.

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/SentinelOne/Integrations/SentinelOne-V2/SentinelOne-V2.py [hunk 1]
```diff
@@ -2624,21 +2624,18 @@ def connect_agent_to_network(client: Client, args: dict) -> Union[CommandResults
     agents_affected = raw_response.get('affected', 0)
 
     # Parse response into context & content entries
-    if agents_affected > 0:
```
Comment (jlevypaloalto) [inline Packs/SentinelOne/Integrations/SentinelOne-V2/SentinelOne-V2.py:2627]: Why are we removing this check?

**Review by jlevypaloalto — COMMENTED** [review]
Hi @chloerongier,
Thank you for your contribution!
Great work :)
Just a short comment:

File: Packs/SentinelOne/Integrations/SentinelOne-V2/SentinelOne-V2.py (hunk 1 above)
Comment (jlevypaloalto) [reply Packs/SentinelOne/Integrations/SentinelOne-V2/SentinelOne-V2.py:2627]: @chloerongier, I believe I didn't explain myself clearly, my apologies.
In the original code the string `No agents were connected to the network.` is returned when no agents are effected. Is there a reason we should change this behavior?

Comment (chloerongier) [reply Packs/SentinelOne/Integrations/SentinelOne-V2/SentinelOne-V2.py:2627]: Indeed, in the original code this string is printed when no agent is affected. But it is only a print, no output **in the context** is returned.

If this command is used with **two** agents IDs as inputs, but only **one** is affected, the command output will still display both agent IDS in the context with their network status each, which will help determine which was indeed affected and which was not.

It makes sense to keep the same logic (context output with the IDs and network status) if no agent at all is affected.

**Comment by jlevypaloalto** [comment]
Hi @chloerongier, the code looks good!

Please resolve the conflicts and then we're ready for a demo. Please check [this page](https://xsoar.pan.dev/docs/contributing/demo-prep), and let me know when you're available for one over DFIR. 
Feel free also to send me a recording of a demo.

**Comment by jlevypaloalto** [comment]
@chloerongier, you can use a video sharing service such as Google Drive or YouTube and provide a link.
When using Google Drive, please make sure I have the permissions to access to the video.

Please feel free to reach out to me with any questions - I'm available here or on slack :)
Thanks again!

**Comment by chloerongier** [comment]
@jlevypaloalto Can you share your email address please ?

**Comment by jlevypaloalto** [comment]
@chloerongier please share the file and I'll send you a request for access.

Please note when merging from master that the location of the `git_pull_master_into_fork.sh` script has moved to `./.circleci/git_pull_master_into_fork.sh`.

**Comment by chloerongier** [comment]
Hello @jlevypaloalto, I'm waiting for my Slack accesses to send you the video. A colleague of mine (username: Pierre-Louis) has requested the access on my behalf already on Slack, could you please acknowledge his request ?

**Review by jlevypaloalto — APPROVED** [review]
Hi @chloerongier, the demo looks great!

Please resolve the conflicts. Note that the location of the `git_pull_master_into_fork.sh `file has changed to `./.circleci/git_pull_master_into_fork.sh.`

**Comment by jlevypaloalto** [comment]
Hi @chloerongier,
I merged the internal PR, and your update is now in the master branch of content repo.
Your update should be available in the marketplace within the next 72 hours.
Again, thank you very much for your contribution!

**Pull request merged**